test/stat_tracker_test.rb

```diff
@@ -198,7 +190,7 @@ def test_it_can_return_total_games_per_team_for_a_season
     assert_equal ({"3"=>5, "6"=>9, "5"=>4, "17"=>1, "16"=>1}), @@stat_tracker.total_games_per_team_for("20122013")
   end
 
```

Comment:
@hopesgit I'm getting an error for this test method: _NoMethodError: undefined method `total_games_per_team_for'_

Reply:
Yeah, I think the method got overwritten in the process of changing a few old helper methods around, so I just pulled it from the current main and put it back in.